code/style.css

```diff
@@ -1,7 +1,490 @@
+  color: #fd99d7;
+  display: none;
+  font-family: 'Kumbh Sans', sans-serif;
+  text-align: center;
+  font-weight: 500;
+  font-size: 1rem;
+}
+
+button {
+  background: none;
+	color: inherit;
+	border: none;
+	padding: 0;
+	font: inherit;
+	cursor: pointer;
+	outline: inherit;
+}
+
+button:active {
+
+  color: red;
+}
+
+
+/* TEXT STYLING */
+
+h1 {
+  flex: 1;
+  font-family: 'Kumbh Sans', sans-serif;
+  margin: auto;
+  width: 100%;  
+  text-align: center;
+  font-size: 3.5rem;
+  font-weight: 600;
+  color: snow;
+  position: absolute;
+  margin-top: 13rem;
+
+}
+
+h2 {
+
+  margin: auto;
+  width: 100%;
+  font-family: 'Kumbh Sans', sans-serif;
+  text-align: center;
+  font-weight: 300;
+  color: snow;
+  position: absolute;
+  padding: 5rem;
+  margin-top: 16rem;
+
+}
+
+h3 {
+  font-family: 'Kumbh Sans', sans-serif;
+  text-align: center;
+  font-weight: 600;
+  font-size: 2rem;
+  color: rgb(252, 208, 252);
+  padding-bottom: 1.5rem;
+}
+
+
+
+
+/* FORM */
+
+#form-section {
+  font-family: 'Kumbh Sans', sans-serif;
+  font-size: 1.3rem;
+  font-weight: 300;
+  color: #fd99d7;
+  padding: 5rem;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  background-image: url("./images/guitar-rainbow.jpg");
+  background-size:cover;
+  background-repeat: no-repeat;
+  background-position: center;
+
+}
+
+.form-box {
+  padding: 3rem;
+  width: 80%;
+  background-color: rgba(27, 28, 77, 0.666);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 4rem;
+  
 }
+
+.signup {
+
+  flex: 1;
+  min-width: 100%;
+}
+
+.right {
+
+  flex: 1;
+  max-width: 400px;
+  min-width: 310px;
+  
+}
+
+.left {
+  flex: 1;
+  max-width: 400px;
+
+}
+
+label {
+
+  padding: 0.1rem;
+}
+
+.right input[type="radio"] {
+
+  padding: 1.2rem;
+  
+}
+
+.right input {
+  margin-top: 0.5rem;
+  margin-bottom: 1rem;
+  padding: 3rem;
+
+}
+
+.left input {
+  width: 100%;
+  height: 50px;
+  margin-top: 0.5rem;
+  margin-bottom: 1rem;
+  border-radius: 5rem;
+
+}
+
+textarea {
+  border-radius: 0.4rem;
+  padding-top: 2rem;
+}
+
+/* Styling for the submit-button */
+
+input[type="submit"] {
+  margin-top: 0.7rem;
+  font-size: 1.2rem;
+	background-color: #fd99d794;
+	border: none;
+  border-radius: 0.4rem;
+	padding: 1rem;
+	font: inherit;
+	cursor: pointer;
+	outline: inherit;
+  width: 100%;
+
+}
+
+.form-box button {
+
+  width: 100%;
+}
+
+/* FOOTER */
+
+footer p {
+  font-family: 'Kumbh Sans', sans-serif;
+  text-align: end;
+  padding-right: 3rem;
+  color: rgb(68, 117, 117);
+  padding-top: 2rem;
+}
+
+
+footer {
+  /* width: 100%; */
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 10rem;
+  background-color: black;
+}
+
+/* MEDIA QUERIES */
+
+    /* MOBILE */
+
+    @media all and (max-width: 569px) {
+
+      #header-section {
+
+        min-height: 350px;
+      }
+
+      video {
+
+        height: 100%;
+
+      }
+
+      h1 {
+        font-size: 2rem;
+        margin-top: 5rem;
+        padding-right: 2rem;;
```

Comment:
here you have 2 ; on 341 and on line 453

Reply:
Thank you for your observant and thorough code review! 🌟